### PR TITLE
feat: GEOT-691 add group header component

### PR DESCRIPTION
**Pull request by mariusz-kabala**

### CODE REVIEW

File: packages/app-andrew/src/components/group/GroupHeader/index.tsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import React, { FC, useState, useEffect } from 'react'
+// state
+import { postsQuery } from '@gtms/state-post'
+// ui
+import { GroupSidebar } from 'components/group/Sidebar'
```
Comment (ra8ga) [inline packages/app-andrew/src/components/group/GroupHeader/index.tsx:5]: can we name it GroupHeader now as it's more on top than left/right

Comment (ra8ga) [reply packages/app-andrew/src/components/group/GroupHeader/index.tsx:5]: I can do this in next PR, should I?

Comment (mariusz-kabala) [reply packages/app-andrew/src/components/group/GroupHeader/index.tsx:5]: yeah, and later you will change the concept and we will rename it. Find a generic name or don't touch it

Comment (ra8ga) [reply packages/app-andrew/src/components/group/GroupHeader/index.tsx:5]: ok let's left if for now - but btw. this concept won't change , this time really , like cyberpunk 10 dec

**Review by ra8ga — APPROVED** [review]

**Pull request merged**